plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduRecordCursor.java

```diff
@@ -58,7 +55,7 @@
     private final List<Type> columnTypes;
     private final KuduTable table;
     private final Map<Integer, Integer> fieldMapping;
-    private RowResultIterator nextRows;
+    private KuduScannerIterator kuduScannerIterator;
```

Comment:
Can you please try to explain to how this change matches: `Send keep alive requests to ensure Kudu scanner does not time out`?

Reply:
Sending keep alive requests is implemted in KuduScannerIterator.